### PR TITLE
Add Porsch project with Nephos

### PR DESCRIPTION
**Pull request by Peter5Lin**
New platform of Pegatron Porsche:
ASIC Vendor: Nephos
Swich ASIC: NP8365
Port Configuration: 10GX48,100GX6
SONiC Image: onie-installer.bin
**- What I did**
Add device and platform files for Pegatron Porsche
**- How I did it**
**- How to verify it**
sensors
show interfaces status
show interfaces transceiver eeprom
show platform syseeprom
show platform psustatus

**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by stcheng** [comment]
retest this please

File: platform/nephos/one-image.mk [hunk 1]
```diff
@@ -6,6 +6,7 @@ $(SONIC_ONE_IMAGE)_IMAGE_TYPE = onie
 $(SONIC_ONE_IMAGE)_INSTALLS += $(NEPHOS_NPS_KERNEL)
 $(SONIC_ONE_IMAGE)_LAZY_INSTALLS += $(INGRASYS_S9130_32X_PLATFORM_MODULE) \
                                $(INGRASYS_S9230_64X_PLATFORM_MODULE) \
-                               $(ACCTON_AS7116_54X_PLATFORM_MODULE)
+                               $(ACCTON_AS7116_54X_PLATFORM_MODULE) \
+			       $(PEGATRON_PORSCHE_PLATFORM_MODULE)
```
Comment (stcheng) [inline platform/nephos/one-image.mk]: use space to indent?

File: device/pegatron/x86_64-pegatron_porsche-r0/porsche/port_config.ini [hunk 2]
```diff
@@ -0,0 +1,55 @@
+# name          lanes		alias			index		speed
```
Comment (stcheng) [inline device/pegatron/x86_64-pegatron_porsche-r0/porsche/port_config.ini]: could you fix the indentation here?

Comment (Peter5Lin) [reply device/pegatron/x86_64-pegatron_porsche-r0/porsche/port_config.ini]: Hi

Please review again, thx

File: platform/nephos/one-image.mk (hunk 1 above)
Comment (Peter5Lin) [reply platform/nephos/one-image.mk]: Hi

Please review again, thx

**Pull request merged**